tasks-by-users/samar4tech/card.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Card-Hover-Effect</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="card">
+        <div class="face face1">
+          <div class="content">
+            <img src="images/design.png" alt="Image1" />
```

Comment:
Alt must describe about the image if it is not rendered ! 
What this `alt="image1"` here ??
@samar4tech